src/interfaces/native/Text.ts

```diff
@@ -43,6 +43,6 @@ export type Marks =
 
 export interface TextProps {
   type: "text"
-  marks: Marks[]
+  marks?: Marks[]
   text: string
```

Comment:
Assuming this is another separate fix that's supposed to go in!

Reply:
Oops yup, the marks was marked as required by accident but a normal text node may be a plain one with no marks being applied.